routes/users/users.js

```diff
@@ -5,6 +5,19 @@ const pool = require('../../db');
 const userRouter = express();
 
 userRouter.use(express.json());
+
+// Get user by id endpoint
+userRouter.get('/users/:id', async (req, res) => {
+  try {
+    const { userId } = req.params;
+    const user = await pool.query('SELECT * FROM users WHERE user_id = $1', [userId]);
```

Comment:
- update param variable name according the previous comment
- the column name for user id in the users table is 'id' in the database (update query to reflect this)